test/integration/runner_task_test.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+require 'kubernetes-deploy/runner_task'
+class RunnerTaskTest < KubernetesDeploy::IntegrationTest
+  def test_works
+    deploy_fixtures("hello-cloud")
+
+    task_runner = KubernetesDeploy::RunnerTask.new(
+      context: KubeclientHelper::MINIKUBE_CONTEXT,
+      namespace: @namespace,
+    )
+
+    task_runner.run(
+      task_template: 'hello-cloud-template-runner',
+      command: nil, # idk
+      args: ['a']
+    )
+
+    assert_logs_match(/Starting task runner/)
+  end
+
+  def test_substitutes_arguments
+    deploy_fixtures("hello-cloud")
+
+    task_runner = KubernetesDeploy::RunnerTask.new(
+      context: KubeclientHelper::MINIKUBE_CONTEXT,
+      namespace: @namespace,
+    )
+
+    pod = task_runner.run(
+      task_template: 'hello-cloud-template-runner',
+      command: nil, # idk
+      args: ['a']
+    )
+
+    assert_equal ['a'], pod.spec.containers.first.args
+  end
+
+  def test_missing_namespace
+    assert_raises(KubernetesDeploy::RunnerTask::FatalTaskRunError) do
```

Comment:
Please add assertions on the actual error messages here, to make sure they are helpful.